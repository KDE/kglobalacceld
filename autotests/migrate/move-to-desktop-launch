[kstuff]
Launch=Meta+K,Meta+K,Launch

[kded5]
Display=Meta+Y,none,Display

[org.kde.bar.desktop]
DoSomething=Meta+Z,Meta+X,Do Something
