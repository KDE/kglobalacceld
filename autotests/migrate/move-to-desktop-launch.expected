[kstuff]
Launch=Meta+K,Meta+K,Launch

[services][org.kde.test.desktop]
_launch=Meta+Y

[org.kde.bar.desktop]
DoSomething=Meta+Z,Meta+X,Do Something
